pkg/yurtctl/constants/yurt-app-manager-tmpl.go

```diff
@@ -668,7 +668,7 @@ spec:
         command:
         - /usr/local/bin/yurt-app-manager
         image: {{.image}}
-        imagePullPolicy: Always
+        imagePullPolicy: IfNotPresent
```

Comment:
yurt-app-manager is recommended to deploy on cloud nodes, so it's better to set imagePullPolicy as `Always` instead of `IfNotPresent`

Reply:
I know you means, but yurt-tunnel-server is also deploy on cloud nodes, it's set imagePullPolicy as IfNotPresent, I just think we need to unify, but that's just a suggestion

Reply:
@JameKeal ok. please `git rebase` the newest master branch to fix e2e test error.

Reply:
OK，finished.